TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
It's fine that the kickstarter as dependency to the NodeTypes package, but please try to keep Neos as decoupled from the NodeType package as possible (if not not completly)

We also start to use raw Neos without the NodeTypes package, and it's a pretty nice practice, that we can promote (or at least not block)